exercises/concept/inventory-management/.docs/hints.md

```diff
@@ -4,32 +4,32 @@
 
 - [The Python Dictionary Tutorial](https://docs.python.org/3/tutorial/datastructures.html#dictionaries) can be a great introduction.
 
-## 1. Create the dictionary
+## 1. Create an inventory based on a list
 
 - You need a [for loop](https://docs.python.org/3/tutorial/controlflow.html#for-statements) to iterate the list of items, then insert each item in the dictionary if missing and increment the item count using the dictionary accessor.
 - You can use [`setdefault`](https://www.w3schools.com/python/ref_dictionary_setdefault.asp) to make sure the value is set before incrementing the count of the item.
 - This function should [return a dict](https://www.w3schools.com/python/ref_keyword_return.asp).
 
-## 2. Add items to a dictionary
+## 2. Add items from a list to an existing dictionary
 
 - You need a [for loop](https://docs.python.org/3/tutorial/controlflow.html#for-statements) to iterate the list of items, then insert each item if not already in the dictionary and [increment](https://www.w3schools.com/python/gloss_python_assignment_operators.asp) the item count using the dictionary accessor.
 - You can use [`setdefault`](https://www.w3schools.com/python/ref_dictionary_setdefault.asp) to make sure the value is set before incrementing the count of the item.
 - The function `add_items` can be used by the `create_inventory` function with an empty dictionary in parameter.
 - This function should [return a dict](https://www.w3schools.com/python/ref_keyword_return.asp).
 
-## 3. Decrement items in the dictionary
+## 3. Decrement items from the inventory
 
 - You need [for loop](https://docs.python.org/3/tutorial/controlflow.html#for-statements) to iterate the list of items, if the number of items is not `0` then [decrement](https://www.w3schools.com/python/gloss_python_assignment_operators.asp) the current number of items.
 - You can use the `key in dict` that returns `True` if the key exists to make sure the value is in the dictionary before decrementing the number of items.
 - This function should [return a dict](https://www.w3schools.com/python/ref_keyword_return.asp).
 
-## 4. Remove an item from a dictionary
+## 4. Remove an item entirely from the inventory
 
 - If item is in the dictionary, [remove it](https://www.w3schools.com/python/ref_dictionary_pop.asp).
 - If item is not in the dictionary, do nothing.
 - This function should [return a dict](https://www.w3schools.com/python/ref_keyword_return.asp).
 
-## 5. List the items that are in stock
+## 5. Return the inventory content
```

Comment:
I like the idea of the change here if I am thinking about the difference of output and return, but returning the inventory content can mean that we are putting it back in inventory or back to where it came from, and might cause some confusion.